staging/src/github.com/labring/lvscare/care/vars.go

```diff
@@ -27,12 +28,31 @@ type LvsCare struct {
 	RealServer    []string
 	RunOnce       bool
 	Clean         bool
+	Test          bool
 	Interval      int32
+	IfaceName     string
 	Logger        string
 	TargetIP      net.IP
-	//
-	lvs   Lvser
-	Route *route.Route
+	// runtime
+	cleanupFuncs []func() error
+	lvs          Lvser
+	Route        *route.Route
+}
+
+func (l *LvsCare) RegisterFlags(fs *pflag.FlagSet) {
+	fs.IPVar(&l.TargetIP, "ip", nil, "target ip")
+	fs.BoolVar(&l.RunOnce, "run-once", false, "is run once mode")
+	fs.StringVar(&l.VirtualServer, "vs", "", "virtual server like 10.54.0.2:6443")
+	fs.StringSliceVar(&l.RealServer, "rs", []string{}, "real server like 192.168.0.2:6443")
+	fs.StringVar(&l.Logger, "logger", "INFO", "logger level: DEBG/INFO")
+	fs.BoolVarP(&l.Clean, "clean", "C", false, "before run clean ipvs rules")
+	fs.BoolVarP(&l.Test, "test", "t", false, "use when any of real server is listening on the same host, "+
```

Comment:
Is it better to delete this parameter? Is it better to default the iface parameter to null?